src/Symfony/Component/ExpressionLanguage/Tests/ExpressionLanguageTest.php

```diff
@@ -12,31 +12,47 @@
 namespace Symfony\Component\ExpressionLanguage\Tests;
 
 use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\ExpressionLanguage\ParsedExpression;
 use Symfony\Component\ExpressionLanguage\Tests\Fixtures\TestProvider;
+use Symfony\Bridge\PhpUnit\ErrorAssert;
 
 class ExpressionLanguageTest extends \PHPUnit_Framework_TestCase
 {
     public function testCachedParse()
     {
-        $cacheMock = $this->getMock('Symfony\Component\ExpressionLanguage\ParserCache\ParserCacheInterface');
+        $cacheMock = $this->getMock('Psr\Cache\CacheItemPoolInterface');
```

Comment:
If it triggers a deprecation notice (in case of wrong argument), it should be labeled with the group Legacy afaik